clients/src/main/java/org/apache/kafka/common/protocol/types/Struct.java

```diff
@@ -70,6 +70,22 @@ public Object get(Field field) {
     }
 
     /**
+     *
+     * @param field
+     * @return
+     */
+    public Object getRaw(Field field) {
```

Comment:
Should this be an `@Override`? i.e. should this be defined in `Type`. If so, then javadoc could move up to that level? Which would probably be good anyway since we kinda need a javadoc here -- it's really not obvious what "raw" means in this context.